components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -821,7 +903,7 @@ func getMergedOpensearchInterface(rawOutput string, pemFilePath string, remoteSe
 		return "", err
 	}
 
-	pemBytes, err := ioutil.ReadFile(pemFilePath) // nosemgrep
+	pemBytes, err := os.ReadFile(pemFilePath) // nosemgrep
```

Comment:
We can remove `nosemgrep` as now we are not using ioutil package. And if possible use `github.com/chef/automate/lib/io/fileutils` package instead of directly using the native package.

Reply:
I will remove the comment, but changing to fileutils will mean to refactor that code, which will impact other flow, which will lead to more testing effort. I would prefer that to be a dedicated refactoring PR for this type of changes.